features/base/navigator/actions.js

```diff
@@ -23,4 +23,4 @@ export function navigate(opts) {
         navigator: opts.navigator,
         room: opts.room
     };
-}
+}
```

Comment:
Hey, I know it's an annoyance but... Could I please ask you to not have such changes in future PRs? I know the very removal of a new line is automatically caused by your editor of choice & not your explicit decision. What I'm asking though is that we don't have files in a PR in which the only change is the removal of the new line at the end. I was told that I should use 'git add --patch ...' to review my modifications before I add them to be committed. Thank you!